tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+from mock import patch
+
+
+class PatchMixin(object):
+    def _patch(self, target, *args, **kwargs):
+        if isinstance(target, str):
+            patcher = patch(target, *args, **kwargs)
+        else:
+            patcher = patch.object(target, *args, **kwargs)
```

Comment:
We have no use of `patch.object` here

Reply:
it is copied as is from drf service setup. I think it can be useful